projects/ngneat/helipopper/src/lib/tippy.directive.ts

```diff
@@ -185,6 +185,12 @@ export class TippyDirective implements OnChanges, AfterViewInit, OnDestroy, OnIn
     return `${this.host.nativeElement.getBoundingClientRect().width}px`;
   }
 
+  private createInstanceOnNonNilValue() {
```

Comment:
There is no need for another method. Add it inside the createInstance()

Reply:
It will be added inside the createInstance() method with the next push.